api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -160,13 +160,21 @@ default AppendFiles newFastAppend() {
   RewriteFiles newRewrite();
 
   /**
-   * Create a new {@link RewriteManifests rewrite manifests API} to replace manifests for this
+   * Create a new {@link RewriteManifests rewrite manifests API} to rewrite manifests for this
    * table and commit.
    *
    * @return a new {@link RewriteManifests}
    */
   RewriteManifests rewriteManifests();
 
+  /**
+   * Create a new {@link ReplaceManifests replace manifests API} to replace manifests for this
+   * table and commit.
+   *
+   * @return a new {@link ReplaceManifests}
+   */
+  ReplaceManifests replaceManifests();
```

Comment:
I'm really reluctant to add another method like this to `Table`, especially when it isn't distinguished from `rewriteManifests`. I'd much rather come up with a way for the current rewrite and manual rewrite to work together, or to use a second level of calls, like this:

```
table.rewriteManifests()
    .manualRewrite()
    .add(...)
    .replace(...)
    .commit();
```

Reply:
But I would prefer to make the configuration methods work at the same time.

Reply:
Having an extra method in `Table` was my original concern. However, I am not sure whether anyone would ever mix direct replacement with rewrite and after @bryanck's suggestion, it seemed like we were putting too much under the same API and that might be confusing to the user. I don't have a strong opinion here. Let me see what it would look like to unite these.